fragments/usage/usage-fragment.md

```diff
@@ -0,0 +1,59 @@
+# Usage Recommendations Specification
+
+The Usage Recommendations section defines conditions under which the model publishers believe the
+model will perform as expected. It is important to note that these are *recommendations* and that
+they **in no way provide any guarantee of model performance under the given conditions**.
+
+If no Usage Recommendation section is provided, then users should assume that the model will only
+perform as expected under conditions that match those described in the [Model Training] section.
+This also applies to any fields that are not present in the Usage Recommendations section. For
+instance, if no `temporal` field is provided, users should assume the model will only perform as
+expected for data in the time range given in the [Model Training] section.
+
+## Usage Recommendation Fields
+
+| Field Name       | Type                     | Description                                                             |
+|------------------|--------------------------|-------------------------------------------------------------------------|
+| spatial          | [Spatial Extent Object]  | The spatial extents within which the model should perform as expected.  |
+| temporal         | [Temporal Extent Object] | The temporal extents within which the model should perform as expected. |
+| description      | string                   | A human-readable description of conditions under which the model should perform as expected. |
```

Comment:
This is a good point @rbavery, I think there are probably a few other properties of the input data that might be useful here as well (GSD for example). Figuring out how exactly to represent these in the spec, could be a bit tricky, though, so my thought was to add these as part of a separate PR.

I'll open an issue to track this and flag you so that you can add input on that.

Reply:
Tracked in #22